doc/index.rst

```diff
@@ -318,6 +318,12 @@ Special typing primitives
       present in a protocol class's :py:term:`method resolution order`. See
       :issue:`245` for some examples.
 
+.. data:: ReadOnly
+
+   See :pep:`705`. Indicates that a :class:`TypedDict` key may not be modified.
```

Comment:
I've been trying to stick to saying that the "entry" is readonly, rather than "key", to avoid possible confusion.

Reply:
The PEP got rewritten to use "item" instead of "entry", as that's what the Python docs/API use elsewhere.